packages/design-system/src/components/Accordion/AccordionItem.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { FunctionComponent, useState } from 'react';
+import classNames from 'classnames';
+import uniqueId from 'lodash.uniqueid';
+
+interface AccordionItemProps {
+  children?: React.ReactNode;
+  /**
+   * Class to be applied to the heading `<h2>` tag of an accordion item.
+   */
+  contentClassName?: string;
+  /**
+   * Class to be applied to the content `<div>` tag of an accordion item.
+   */
+  headingClassName?: string;
+  /**
+   * Boolean to expand the accordion.
+   */
+  expanded?: boolean;
+  /**
+   * Text for the accordion item heading.
+   */
+  heading: React.ReactNode | string;
+  /**
+   *  If not provided, a unique id will be automatically generated and used.
+   */
+  id?: string;
+}
+
+const AccordionItem: FunctionComponent<AccordionItemProps> = ({
+  children,
+  contentClassName,
+  headingClassName,
+  expanded,
+  heading,
+  id,
+}) => {
+  // Set the state for opening and closing an accordion item
+  const [open, setOpen] = useState(!!expanded);
```

Comment:
I dont think we can use React Hooks just yet, there are certain apps that are on older versions of react

Reply:
@line47 @nichia 
This can be handled in a follow up, but I think we may need to think more on how we handle states in this component. The way it's currently setup only allows for the "multiselect" option, when ideally we can support the single select by default. We probably also want to allow users to manually control individual accordion items, or at least get access to the click events via an event handler (similar to `onOpen` and `onClose` in Tooltips).

There are a few different options here:
1. Make `AccordionItem` strictly a controlled component by adding required `open` and `onClick` props. Then, make the `Accordion` component handle the state for each `AccordionItem` by default, and allow users to select the Accordion behavior, either "single select", "multi select", or "uncontrolled". This allows us to support single/multi select out of the box, while also providing an uncontrolled option if users need custom behavior
2. We continue to handle state inside `AccordionItem` by default, but we add optional `open` and `onClick` props. This way teams that need single select can manually achieve this behavior by overriding the default behavior (similar to `value` and `defaultValue` in form components). The downside to this is that teams need to do more work to achieve a single select behavior, which is the default for USWDS.

Reply:
Ah, I realized there's already an `expanded` prop. However, that is only used on initializing the component, so we still need to handle the issue i mentioned above